tests/1/webpack.config.ts

```diff
@@ -1,7 +1,11 @@
 import {provideConfiguration} from "../../index";
 import {WebpackOptions} from "webpack/declarations/WebpackOptions";
+import * as path from "path";
 
 export default provideConfiguration({
+	entry: './www/src/index.js',
+	devServer: '/www/dist/',
+	distPath: path.resolve(__dirname+ '/www', 'dist'), // must be here due to usage in different projects
```

Comment:
I do not like this config options as it is not descriptive enough for people who are not familiar with WebPack. What about just having entryPoint: 'www/src/index.js', outputDirectory: 'www/dist', rootPath: path.resolve(__dirname)

Reply:
Ok.